senders/client_test.go

```diff
@@ -58,6 +58,13 @@ func TestSendDirect(t *testing.T) {
 	doTest(t, wf)
 }
 
+func TestSendDirectWithTags(t *testing.T) {
+	tags := map[string]string{"foo": "bar"}
+	wf, err := senders.NewSender("http://"+token+"@localhost:"+wfPort, senders.SDKMetricsTags(tags))
```

Comment:
Just curious what this is testing.  This line uses "foo": "bar" as the SDK metric tags, but I don't see where this is verified.  I mean you could leave `senders.SDKMetricsTags(tags)` out and the test would still pass.

Reply:
Travis, I wanted to make sure that the WFSender is still able to send spans & metrics after we add the SDKMetricsTag, But you are correct, the unit test is missing an assertion to check if these SDKMetricsTags are set. I am looking into options to assert it.